code/index.html

```diff
@@ -4,27 +4,77 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
+<link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <!-- video or image as a header is cool :) -->
+    <header class="hero-header">
+      <picture>
+        <source srcset="./images/heromsmall.jpg" media="(max-width: 667px)">
+        <source srcset="./images/heromedium.jpg" media="(max-width: 1024px)">
+        <img class="hero-image" src="./images/herobig.jpg" alt="Hero image"> 
+      </picture>
+        <h1 class="tenko">Tenko Museum of Modern Art</h1>
+    </header>
 
-    <!-- Signup form -->
+    <div class="both"> 
+    <section class="faq-section">
+      <h1 class="faq-topic">FAQS</h1>
+    
+    <div class="accordion">
+      <h2 class="header1" id="header1"> 
+        If I want to visit the café, do I need to buy an entrance ticket?</h2>
+        <p class= "content1" id="content1">
+          Yes, you will need to buy an entrance ticket to visit the café.</p>
+    </div>
 
-    <section class="accordion">
-      <!-- Accordion -->
-      <h1>FAQ</h1>
+    <div class="accordion">
+        <h2 class="header2" id="header2"> 
+          Can I bring my own food?</h2>
+        <p class= "content2" id="content2">
+          If you bring food from outside the museum, you need to eat it in the park.  </p>
+    </div>
 
-      <div class="question" id="section1">Section 1</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
-      </div>
+    <div class="accordion">
+        <h2 class="header3" id="header3"> 
+          Is it allowed to take photos?</h2>
+        <p class= "content3" id="content3">
+          It is generally allowed to photograph without the use of flash.</p>
+    </div>
 
-      <div class="question" id="section2">Section 2</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
-      </div>
+    <div class="accordion">
+        <h2 class="header4" id="header4"> 
+          Can I bring my dog?</h2>
+        <p class= "content4" id="content4">
+          Animals are not allowed in the park or in the galleries.</p>
+    </div>
     </section>
-    <!-- dont forget to link the javascript file here! -->
+
+    <div class="form-container">
+    <form action="https://httpbin.org/anything" method="post">
+      <h1 class="form-topic">Get the latest Updates - <br> Subscribe below!</h1>
+      <input name="Name" type="text" placeholder="Name">
+      <input name="E-mail" type="text" placeholder="Email">
+      <input name="Number" type="Phone-number" placeholder="Phone-number">
+
+    <div class="terms">
+      <p>Accept terms and conditions</p>
+      <input type="checkbox">
+    </div>
+
+      <label for="Male">Male</label>
+      <input type="radio">
+
+      <label for="Female">Female</label>
+      <input type="radio">
+
+      <label for="Genderqueer/Non-Binary">Genderqueer/Non-Binary</label>
+      <input type="radio" >
```

Comment:
Radio buttons need the `name` attribute in order to work together as a group. Read more in the general feedback document. Also, when you have the label outside of the input, you need to add a corresponding `id` to the input